compose/ui/ui-text/src/desktopMain/kotlin/androidx/compose/ui/text/platform/DesktopFont.desktop.kt

```diff
@@ -201,7 +206,6 @@ internal actual fun loadTypeface(font: Font): SkTypeface {
     }
     val typeface = when (font) {
         is ResourceFont -> typefaceResource(font.name)
-        // TODO: replace with FontMgr.makeFromFile(font.file.toString())
```

Comment:
I removed this TODO I left here a couple years ago as it seems it was addressed in the line below. Someone just forgot to remove the TODO :)